### ISSUE TITLE
Ability/checkbox to "use default value" for image attributes

### ISSUE THREAD

**Issue by IanFiretoys** [magento/magento2#29050]
### Description (*)

As image attributes are rarely set to global scopes, they can become split off from default storeviews very easily (and for good reason, translation issues between multi-language sites, imperial vs metric measurements on swatches etc.) however once they are split there is no way to put them back short of deleting the row for certain attribute IDs from the database directly. this is a) risky and b) severely limits the number of people within any organisation that can then change the images efficiently. 

A checkbox for "Use default values" (as any other non-global scope attribute would get) in the image detail window, or in the image attribute block within a product. would make a huge difference.

### Expected behavior (*)

"use default values" checkbox would remove the lines from the database for any attributes that image is currently using from that storeview. this may need to be for all images at once, as the attributes are often spread between multiple images, or grouped together on one.

### Benefits

it would help stop database creep, as adding new rows for every image attribute on every storeview across a large catalog adds thousands of unnecessary rows. would also make changing image attributes more efficient once they have been changed, as you don't have to go through every storeview each time to make changes across the catalog

it would also give an immediate visual indicator of whether image attributes have already been split off, as currently there is no way of knowing which attributes are using defaults or not, meaning if you get the generic error message of "cannot delete image, it is being used in another store" the only way you can find out which store has its own entries is via mysql query

### Additional information

some mockups of the 2 different implementations that i can see working.

one would be easier to implement than the other, but offers less versatility

![usedefault-indiv](https://user-images.githubusercontent.com/33158810/87014709-6a561a80-c1c4-11ea-90b1-0a2293e5a112.jpg)
![not-usedefault-indiv](https://user-images.githubusercontent.com/33158810/87014741-7510af80-c1c4-11ea-92d4-154d25c21cf3.jpg)

![usedefault-all](https://user-images.githubusercontent.com/33158810/87014757-7a6dfa00-c1c4-11ea-9917-8342ff04dba7.jpg)
![not-usedefault-all](https://user-images.githubusercontent.com/33158810/87014874-a5f0e480-c1c4-11ea-9960-d8a44b72446c.jpg)

**Comment by erfanimani** [comment]
I would love to have this feature. We're running into the same issue.

+1

**Comment by nicolai-hoeks** [comment]
Very much still relevant. 
Managing images is a nightmare when managing multiple storeviews

**Comment by IanFiretoys** [comment]
still definitely relevant, I'm having to reset images through the database on a regular basis still, which is not ideal. I get several emails a month from our non technical staff saying that they can't change/delete an image because it's got a tag selected in another storeview.

**Comment by nicolai-hoeks** [comment]
Still very relevant
It is a nightmare to manage images once a single change on images has been made to an image on a store-view

**Comment by mts527** [comment]
Any news regarding this feature? It would make administrating images with multiple stores a lot easier.

**Comment by IanFiretoys** [comment]
yeah, can this be made a slightly higher priority now that #34169 has been confirmed to be affecting core magento? there will be a lot of people out there with split off image attributes they don't know about or know how to fix without googling "The image cannot be removed as it has been assigned to the other image role" and digging in their database.

**Comment by Dan-Shirley** [comment]
Has this moved forward at all?

This issue is extremely frustrating, I have multiple people helping to add/update images on our multisite and I get asked over and over "why are the images not updating?", "why can't I delete this image" it's always store view related. Having to update the same product image in multiple store views is frustrating and a huge waste of time.

We had a 'use default' option in M1 (not sure if this was native or added by a dev at some point) it worked, it was simple.

**Comment by vseager** [comment]
This needs to be done, it is a major usability issue

**Comment by engcom-Bravo** [comment]
Hi @IanFiretoys,

Thanks for your reporting and collaboration.

 We gone through with the description and seems to be good to have feature to proceed further Confirming the issue.

Thanks.
